src/ui/handler/scroll_zoom.js

```diff
@@ -4,9 +4,23 @@ const DOM = require('../../util/dom');
 const util = require('../../util/util');
 const browser = require('../../util/browser');
 const window = require('../../util/window');
+const interpolate = require('../../style-spec/util/interpolate').number;
+const LngLat = require('../../geo/lng_lat');
 
 import type Map from '../map';
 import type Point from '@mapbox/point-geometry';
+import type Transform from '../../geo/transform';
+
+// deltaY value for mouse scroll wheel identification
+const wheelZoomDelta = 4.000244140625;
+// These magic numbers control the rate of zoom. Trackpad events fire at a greater
+// frequency than mouse scroll wheel, so reduce the zoom rate per wheel tick
+const defaultZoomRate = 1 / 100;
+const wheelZoomRate = 1 / 500;
```

Comment:
I think I prefer the faster scrolling of `400` here, but `500` works well too.

Reply:
I split the difference 😄